presto-main/src/main/java/com/facebook/presto/server/remotetask/RemoteTaskStats.java

```diff
@@ -72,9 +55,10 @@ public void updateWithoutPlanSize(long bytes)
     }
 
     @Managed
-    public double getResponseSizeBytes()
+    @Nested
```

Comment:
Should we use `@Flatten` instead of `@Nested` ? That would ensure that the metrics exported remain in the same format.

Reply:
Good catch.